api/src/pcapi/scripts/jira_26180_fix_rpoints.py

```diff
@@ -0,0 +1,145 @@
+            comment = COMMENT_WITH_BANK_INFO.format(
+                venue_id=venue.id,
+                reimbursement_point_id=reimbursement_point_id,
+                bank_info_id=bank_info.id,
+                dms_id=bank_info.applicationId,
+                end_date=end_date,
+            )
+        else:
+            comment = COMMENT_WITHOUT_BANK_INFO.format(
+                venue_id=venue.id,
+                reimbursement_point_id=reimbursement_point_id,
+            )
+
+        rpoint_link = venue.current_reimbursement_point_link
+        assert rpoint_link.reimbursementPointId == reimbursement_point_id
+        rpoint_link.timespan = db_utils.make_timerange(
+            rpoint_link.timespan.lower,
+            # In some cases (or maybe all?), the link was created just
+            # after the last modification time of the BankInfo. So the
+            # lower bound of the timespan is slightly _after_ the
+            # modification time. In that cas, we cannot use the
+            # modification time as the upper bound (because it must be
+            # after the lower bound), so we use the lower bound instead.
+            # Also, add 1 second. Otherwise, if the upper bound has the
+            # same value as the lower bound, the resulting tsrange object
+            # is empty because the lower bound is inclusive but the upper
+            # bound is not.
+            max(end_date, rpoint_link.timespan.lower) + datetime.timedelta(seconds=1),
+        )
+        db.session.flush()
+        action = history_api.log_action(
+            action_type=history_models.ActionType.LINK_VENUE_BANK_ACCOUNT_DEPRECATED,
+            author=USER,
+            offerer=venue.managingOfferer,
+            venue=venue,
+            comment=comment,
+            save=False,
+        )
+        db.session.add(action)
+        db.session.flush()
+        print(f"Reimbursement point {reimbursement_point_id}: ended rpoint link {rpoint_link.id} as of {end_date}")
+
+
+def get_reimbursement_points_to_fix() -> list[int]:
+    """Return ids of reimbursement points that have a non-accepted
+    BankInformation (or no BankInformation)
+    """
+    query = """
+      with rpoint as (
+        select distinct("reimbursementPointId") as id from venue_reimbursement_point_link
+        where upper(timespan) is null
+      )
+      select distinct(rpoint.id) from rpoint
+      left outer join bank_information on bank_information."venueId" = rpoint.id
+      left outer join cashflow ON cashflow."reimbursementPointId" = rpoint.id
+      where
+        (
+          bank_information.status is null
+          or bank_information.status != 'ACCEPTED'
+        )
+        and cashflow.id is null
+        order by 1
+    """
+    result = db.session.execute(query)
+    rows = result.fetchall()
+    return sorted(row[0] for row in rows)
+
+
+def fix_all(save: bool) -> None:
+    if save:
+        print("THIS IS REAL, you have 5 seconds to interrupt.")
```

Comment:
J'aime l'idée, et on pourrait même faire:
```suggestion
        from pcapi import settings
        (...)
        print(f"THIS IS REAL, and we are in {settings.ENV} /n You have 5 seconds to interrupt.")
```